### DISCUSSION TITLE
Startup Code and standalone Exe

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1392] [Q&A] [answered]
Hello,

Where can I find the startup code (Boot) of Fprime Software and also the final exe generated.

I assume for default build in Ref it was in 

> Ref/build-artifacts/Linux/bin/Ref

Is it correct ?

**Comment by timcanham** [comment, marked as answer]
That is correct. The `main()` entry point is in `Ref/Top/Main.cpp`.
